src/node_buffer.cc

```diff
@@ -675,6 +675,48 @@ void Fill(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+void StringWriteWithoutBuffer(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = args.GetIsolate();
+
+  if (!args[0]->IsString())
+    return env->ThrowTypeError("Source must be a string");
+  enum encoding source_encoding = ParseEncoding(isolate, args[1], UTF8);
+  enum encoding target_encoding = ParseEncoding(isolate, args[2], UTF8);
+
+  Local<String> str = args[0]->ToString(isolate);
```

Comment:
You can use `args[0].As<String>()` here.  Slightly faster, it's just a `reinterpret_cast` under the hood.

(Also, the version of `ToString()` that takes an isolate is deprecated.)